lang/test/org/partiql/lang/syntax/SqlParserTest.kt

```diff
@@ -3987,6 +3987,13 @@ class SqlParserTest : SqlParserTestBase() {
         """
     )
 
+    @Test
```

Comment:
Sorry if this wasn't documented, but for new tests in this file, we should be using the [assertExpression](https://github.com/partiql/partiql-lang-kotlin/blob/0dee7a6ce31c5482b5769f6507aa32006c923ae9/lang/test/org/partiql/lang/syntax/SqlParserTestBase.kt#L110) that asserts on the `PartiqlAst` rather than the s-expression representation of the AST. The s-expression version of the AST can be harder to read and directly interpret compared to the PIG-builder.

Could the tests from L3990-4040 use the `assertExpression` that asserts on the PIG-builder? Example test using the PIG-builder `assertExpression` https://github.com/partiql/partiql-lang-kotlin/blob/0dee7a6ce31c5482b5769f6507aa32006c923ae9/lang/test/org/partiql/lang/syntax/SqlParserTest.kt#L1793-L1804